internal/impl/aws/input_kinesis.go

```diff
@@ -750,8 +750,7 @@ func (k *kinesisReader) stealShard(wg *sync.WaitGroup, streamID string, clientCl
 	return errShardNotStolen
 }
 
-//------------------------------------------------------------------------------
-
+// ------------------------------------------------------------------------------
```

Comment:
nit: could we please revert this to not to touch unnecessary files in Benthos

Reply:
yeah i forgot to revert. hahaha thanks